### DISCUSSION TITLE
Components not displaying/working

### DISCUSSION THREAD

**Discussion by ArmantHitchcock** [primefaces/community#1155] [PrimeVue] [answered]
Hi, there.

Came across PrimeVue and trying it out, work prefers to use Vue over a cdn rather than Nuxt or Vite.
So, I'm trying to get some components to work but I'm not getting any traceback errors on the console.
Only 2 works so far, well sort of the calendar works but I'm struggling the bind the placeholder, 
and the checkbox works but there is a checkmark icon that appears bellow it (not sure why, was expecting true or false). 
The rest of the components aren't working at all.
Unfortunately, I'm not finding much on documentation other than an example code [https://primevue.org/cdn/](url) and the video.

the code I'm using:
index.html
[https://codepen.io/Armant/pen/ExMebgE](url)

I'm at a loss right now and don't know what else to try, any guidance would be greatly appreciated.

**Comment by tugcekucukoglu** [comment, marked as answer]
I cannot open the demo but I want to share a working demo: https://stackblitz.com/edit/web-platform-qq5ahr?file=index.html
